.github/workflows/deploy.yml

```diff
@@ -24,6 +24,14 @@ jobs:
           CF_ZONE_ID: ${{ secrets.CF_ZONE_ID }}
           CF_DEPLOY_DOMAIN: ${{ secrets.CF_DEPLOY_DOMAIN }}
 
+      # this is needed to get the lowercase version of the actor name
+      # TODO: switch to some lowercase function in the future when Actions supports it
+      - name: Set lowercase actor name
+        run: |
+          echo "ACTOR_LOWER=${GH_ACTOR,,}" >> ${GITHUB_ENV}
```

Comment:
The change makes sense to me. However this bash syntax is terriblee, we could use this instead: `| awk '{ print tolower($0) }'`